compiler/rustc_session/src/lint/builtin.rs

```diff
@@ -2555,6 +2555,13 @@ declare_lint! {
     @feature_gate = sym::unsafe_block_in_unsafe_fn;
 }
 
+declare_lint! {
+    //FIXME: Add explanation.
+    pub FALL_BACK_TO_NEVER_TYPE,
+    Deny,
+    "Unresolved variable might fall back to never_type `!`"
```

Comment:
Nitpick: the style for user visible messages is _not_ to have leading upper-case titling.
```suggestion
    "unresolved variable might fall back to never_type `!`"
```

Reply:
Thanks a lot @estebank I will make the change.